starter-code/css/main.css

```diff
@@ -4,7 +4,151 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
+html {
```

Comment:
If you're ever styling the entire html element, it's best to limit only to settings such as width or height. It looks like you're trying to use this to create the blue background color, which doesn't appear.